packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -1420,7 +1423,7 @@ function createIdealGraph(
             target: firstSourceBundle.target,
             type: firstSourceBundle.type,
             env: firstSourceBundle.env,
-            manualSharedBundle: manualSharedObject.name,
+            manualSharedBundle: manualSharedObject.name + i,
```

Comment:
Not sure we should be changing the `manualSharedBundle` property on the bundle as that will give unexpected results in naming. Could we add the index to the unique key without polluting this property?

Reply:
Not sure I understand the issue with this. This is only for split vendor bundles, which a user can't specify unique names for through the config anyway. So in this case it'd be like `shared~vendor`, `shared~vendor1` , `shared~vendor2`. I thought we wanted the names **and** the unique key to be unique ?

Reply:
Otherwise if the `manualSharedBundle.name` is not unique, then maybe we should go back to setting the `uniqueKey` to asset ids ?